tests/analysis/tests.bzl

```diff
@@ -97,14 +92,14 @@ action_full_contents_test = analysistest.make(_action_full_contents_test_impl)
 def _test_action_full_contents():
     detekt(
         name = "test_target_full",
-        srcs = ["path A.kt", "path B.kt", "path C.kt"],
-        cfgs = ["config A.yml", "config B.yml", "config C.yml"],
+        srcs = ["A.kt"],
+        cfgs = ["A.yml"],
         baseline = "baseline.xml",
         html_report = True,
         xml_report = True,
         build_upon_default_config = True,
         disable_default_rulesets = True,
-        fail_fast = True,
+        fail_fast = False,
```

Comment:
Having this set to true fails the build when these targets are evaluated for testing.